fs/fs_zenfs.cc

```diff
@@ -1081,6 +1088,8 @@ Status NewZenFS(FileSystem** fs, const std::string& bdevname) {
           zbd_status.ToString().c_str());
     return Status::IOError(zbd_status.ToString());
   }
+  
+  zbd->SetMetrics(std::make_shared<NoZenFSMetrics>());
```

Comment:
Metrics are already set to `NoZenFSMetrics` during construction of `zbd`.

Reply:
Yes, this statement is more of a usage suggestion, i.e. "where you should stuff your own implementation of Metrics". Would it be better to remove it?

Reply:
I think so.